hawc/apps/study/views.py

```diff
@@ -106,6 +106,17 @@ def post_object_save(self, form):
         self.object.searches.add(search)
 
 
+class IdentifierStudyCreate(ReferenceStudyCreate):
```

Comment:
swapped the form and update page out to the standard  `ReferenceStudyCreate` instead of a hand-rolled one